gcloud-java-examples/src/main/java/com/google/gcloud/examples/Stat.java

```diff
@@ -0,0 +1,108 @@
+package com.google.gcloud.examples;
+
+import java.io.IOException;
+import java.net.URI;
+import java.nio.file.FileSystem;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.spi.FileSystemProvider;
+
+/**
+ * Stat is a super-simple program that just displays the size of the file
+ * passed as argument.
+ *
+ * <p>It's meant to be used to test GCloud's integration with Java NIO.
+ *
+ * <p>You can either use the '--check' argument to see whether GCS is enabled,
+ * or you can directly pass in a GCS file name to use. In that case you have to
+ * be logged in (using e.g. the gcloud auth command).
+ *
+ * <p>See the README for a command line to run this example.
+ */
+public class Stat {
+
+  /**
+   * See the class documentation.
+   */
+  public static void main(String[] args) throws IOException {
+    if (args.length == 0) {
+      System.out.println("Use --help for help or provide a file name.");
```

Comment:
Nit: since there are more options other than just providing a file name, you could just say "Use --help for help", which will print out all options (including --list, --check, and filenames)